src/Makefile.am

```diff
@@ -677,13 +677,13 @@ FORCE:
 # Otherwise, set it from the contents of .git_version.
 
 ceph_ver.h: .git_version
-	if [ -n "$$NO_VERSION" ] ; then \
-	    if [ ! -f ./ceph_ver.h ] ; then \
-	        $(srcdir)/make_version -n ./ceph_ver.h ; \
-	    fi; \
-        else \
-	    $(srcdir)/make_version $(srcdir)/.git_version ./ceph_ver.h ; \
-	fi
+if NO_VERSION
```

Comment:
Could you implement this so that it sets the NO_VERSION environment variable by default instead ? Otherwise it will be confusing to people who already rely on NO_VERSION=true environment variable to do the same.